.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
i suspect this won't be accepted upstream and recommend putting it in your ~/.gitignore. we can try though

Reply:
Removed.

Also I think it won't be accepted upstream because I did bypass the `actions-js-toolkit` library. I didn't know how to test my use of octokit `3.5.2` easily otherwise (the lib uses `1.2.5` so don't have all the wished features).